docs/use-cases/Veterans.md

```diff
@@ -0,0 +1,85 @@
+# Veterans enrollment pathway
+
+## Overview
+
+This use case describes a feature in the [Cal-ITP Benefits app](https://benefits.calitp.org) that allows US veterans who use public transit to verify their veteran status and receive reduced fares when paying by contactless debit or credit card at participating transit providers in California.
+
+**Actor:** A US veteran who uses public transit in California. For benefit eligibility, a veteran is defined as “a person who served in the active military, naval, or air service, and was discharged or released therefrom under conditions other than dishonorable.” ([source](https://www.ssa.gov/OP_Home/comp2/D-USC-38.html))
+
+**Goal:** To verify a transit rider’s veteran status and enable the rider to receive reduced fares when paying by contactless debit or credit card.
+
+**Precondition:** The California transit provider delivering fixed route service has installed and tested validator hardware necessary to collect fares using contactless payment on bus or rail lines, and the provider has a policy to offer a transit discount for US veterans.
+
+## **Basic Flow**
```

Comment:
I think these bold marks `**` aren't needed on headings.

Reply:
I agree. That styling came with the export from Clickup. Good catch! I've removed the bolding in a new commit.